### PR TITLE
Update configure_etcd.md

### PR DESCRIPTION
**Pull request by arthuo**
Translate the English part of the document

### CODE REVIEW

File: site/zh-CN/reference/sys_config/configure_etcd.md [hunk 1]
```diff
@@ -86,16 +86,16 @@ Under this section, you can configure etcd endpoints, relevant key prefixes, etc
...
   </thead>
   <tbody>
     <tr>
       <td>
-        <li>Sub-prefix of the key to where Milvus stores timestamps in etcd.</li>
-        <li>Caution: Changing this parameter after using Milvus for a period of time will affect your access to old data.</li>
-        <li>It is recommended not to change this parameter if there is no specific reason.</li>
+        <li>Milvus 在 etcd 中存储时间戳的键的子前缀。</li>
+        <li>注意：使用 Milvus 一段时间后更改此参数会影响对旧数据的访问。</li>
+        <li>如果没有具体原因，建议不要更改此参数。</li>
```
Comment (LocoRichard) [inline site/zh-CN/reference/sys_config/configure_etcd.md:98]: “如无特殊原因”

**Review by LocoRichard — CHANGES_REQUESTED** [review]

**Comment by LocoRichard** [comment]
Hi @arthuo , thank you for your contribution, I have made necessary feedbacks.
Sorry for the waiting. I had a day-off on Friday.

**Comment by arthuo** [comment]
> Translate the English part of the document

Thank you for your necessary feedback. By the way, I also want to take a vacation. But today is Monday.

**Comment by LocoRichard** [comment]
Why Not? Haha. You need to give yourself a break from time to time. Don't push yourself too hard. 
Merging the PR.

**Pull request merged**